proxy/http_headers.go

```diff
@@ -72,6 +73,20 @@ func addHeaders(r *http.Request, cfg config.Proxy) error {
 	return nil
 }
 
+// addRequestID Set header with name from `cfg.RequestID` to UUID value
+//
+func addRequestID(r *http.Request, p *HTTPProxy) {
+	if p.Config.RequestID != "" {
```

Comment:
I have did it in separate function because it need HTTPProxy, which is not available in addHeaders.
Changing addHeaders signature lead to massive change of it unit tests.